versioned/persist/adapter/src/main/java/org/projectnessie/versioned/persist/adapter/ReferencesUtil.java

```diff
@@ -1,191 +0,0 @@
-/*
- * Copyright (C) 2022 Dremio
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.projectnessie.versioned.persist.adapter;
-
-import com.google.common.annotations.Beta;
-import com.google.protobuf.ByteString;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Set;
-import java.util.stream.Stream;
-import org.agrona.collections.Hashing;
-import org.agrona.collections.Object2ObjectHashMap;
-import org.agrona.collections.ObjectHashSet;
-import org.projectnessie.versioned.GetNamedRefsParams;
-import org.projectnessie.versioned.Hash;
-import org.projectnessie.versioned.NamedRef;
-import org.projectnessie.versioned.ReferenceInfo;
-import org.projectnessie.versioned.ReferenceNotFoundException;
-
-@Beta
-public final class ReferencesUtil {
```

Comment:
This refactoring isn't necessary and should be reverted

Reply:
At `PersistVersionStore`, to call `ReferenceUtils`, I need `databaseAdapterConfig`
But it is hard to extract config from `TracingDatabaseAdapter`.
So, I moved those methods from 'ReferenceUtils' to `AbstractDatabaseAdapter`

Do you want me to add a method to `DatabaseAdapter` interface to get the configuration? I feel current refactoring makes these methods to be inlined with other methods.